cla_public/translations/cy/LC_MESSAGES/messages.po

```diff
@@ -3349,10 +3310,117 @@ msgid ""
 "If you’re applying for legal aid for a mental health issue,\n"
 "          the requirements for the financial assessment are less rigorous than\n"
 "          for other legal aid problems."
+msgstr "Os ydych yn gwneud cais am gymorth cyfreithiol ar gyfer problem iechyd meddwl, nid yw’r gofynion ar gyfer yr asesiad ariannol mor llym o’i gymharu â phroblemau cymorth cyfreithiol eraill."
+
+#: cla_public/templates/checker/result/ineligible.html:3
+msgid "Help organisations"
+msgstr ""
+
+#: cla_public/templates/checker/result/ineligible.html:8
+msgid "you and your partner"
+msgstr ""
+
+#: cla_public/templates/checker/result/ineligible.html:8
+msgid "you"
+msgstr ""
+
+#: cla_public/templates/checker/result/ineligible.html:9
+msgid "you and your partner’s"
+msgstr ""
+
+#: cla_public/templates/checker/result/ineligible.html:9
+msgid "your"
+msgstr ""
+
+#: cla_public/templates/checker/result/ineligible.html:14
+msgid "You’re unlikely to get legal aid"
+msgstr ""
+
+#: cla_public/templates/checker/result/ineligible.html:17
+#, python-format
+msgid ""
+"From\n"
+"    what you have told us today it looks like you won’t be able to get\n"
+"    legal aid %(fromcla)s as you don’t qualify financially"
+msgstr ""
+
+#: cla_public/templates/checker/result/ineligible.html:17
+msgid "from CLA"
+msgstr ""
+
+#: cla_public/templates/checker/result/ineligible.html:24
+msgid "This is because"
+msgstr ""
+
+#: cla_public/templates/checker/result/ineligible.html:26
+msgid ""
+"have too much disposable\n"
+"      capital"
+msgstr ""
+
+#: cla_public/templates/checker/result/ineligible.html:31
+msgid ""
+"gross income is more than\n"
+"      the maximum allowed"
+msgstr ""
+
+#: cla_public/templates/checker/result/ineligible.html:36
+msgid ""
+"disposable income is\n"
+"      higher than the limit we allow"
+msgstr ""
+
+#: cla_public/templates/checker/result/ineligible.html:43
+msgid ""
+"Disposable capital includes savings, valuable items and the\n"
+"      equity in any property you own."
+msgstr ""
+
+#: cla_public/templates/checker/result/ineligible.html:48
+msgid ""
+"Disposable income is the money you have left after we’ve accounted\n"
+"      for certain living expenses, like rent or mortgage payments."
+msgstr ""
+
+#: cla_public/templates/checker/result/ineligible.html:57
+msgid "Family Mediation Council"
+msgstr ""
+
+#: cla_public/templates/checker/result/ineligible.html:57
+#, python-format
+msgid ""
+"\n"
+"        If you want to make an application to court about a family matter you need to first of all\n"
+"        see if family mediation will help. Use the %(family_mediation_link)s directory to find a mediator\n"
+"        and make an appointment for a Mediation Information and Assessment Meeting (MIAM).\n"
+"      "
 msgstr ""
```

Comment:
Another one of missing translation

Reply:
This translation that was deleted is not the translation of this phrase.  One for round 3 I think.